### ISSUE TITLE
Too low contrast between background and foreground

### ISSUE THREAD

**Issue by torgeirl** [devilry/devilry-django#1109]
Success Criteria 1.4.3 in WCAG 2 (AA - must) fails due to too low contrast between background and foreground.

Failing elements on front page:
```HTML
<p class="django-cradmin-listbuilder-itemvalue-titledescription-description"> Upload deliveries or see your delivery and feedback history. </p>
<p class="django-cradmin-listbuilder-itemvalue-titledescription-description"> Manage departments, courses, semesters and assignments. </p>
```

**Comment by Eskalol** [comment]
Also fails for admin, tuned color variables slightly. @StianJul  could you check if you agree with the changes?

**Comment by StianJul** [comment]
Using the chrome extensions `WCAG Color contrast checker` and `Color Contrast Analyzer` it now seems to be within the AA requirement.

Closing this issue, but this might not be 100% everywhere, and is probably a much bigger task since we would have to analyze every view and take into consideration that we might need to make major changes to the core of the theme.
